programs/drift/src/state/order_params.rs

```diff
@@ -402,31 +429,27 @@ impl OrderParams {
                     .last_oracle_price_twap_5min,
             )?;
 
+        let frac_of_long_spread_in_price: i64 = perp_market
```

Comment:
can save some compute by doing the calculation in the match blocks

Reply:
i need both of them for both though

for the slow, i pushing it up slightly from the opposite side of the book
for the fast, im pulling it away from the mid